File: src/Python/705.DesignHashSet.py
class MyHashSet:

    def __init__(self):
        """
        Initialize your data structure here.
        """
        self.bucket = [0] * 1000000

    def add(self, key):
        """
        :type key: int
        :rtype: void
        """
        self.bucket[key - 1] = 1

    def remove(self, key):
        """
        :type key: int
        :rtype: void
        """
        self.bucket[key - 1] = 0

    def contains(self, key):
        """
        Returns true if this set contains the specified element
        :type key: int
        :rtype: bool
        """
        return self.bucket[key - 1] == 1

        

File: src/CSharp/705.DesignHashSet.cs
using System;
using System.Collections;
using System.Collections.Generic;

namespace PeefyLeetCode.BinarySearch
{
public class MyHashSet {
    HashSet<int> hash;
    /** Initialize your data structure here. */
    public MyHashSet() {
        hash = new HashSet<int>();
    }
    
    public void Add(int key) {
        hash.Add(key);
    }
    
    public void Remove(int key) {
        hash.Remove(key);
    }
    
    /** Returns true if this set contains the specified element */
    public bool Contains(int key) {
        return hash.Contains(key);
    }
}
}
